<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificationType</class>
 <widget class="QWidget" name="ModificationType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="gridFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>25</number>
      </property>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color:#EBEBEB;
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="topMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLineEdit" name="txfRecherche">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	border-style:none;
	border-radius: 12px;
	background-color: #B9B9B9;
	padding-left:20px;
	color:black;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Rechercher...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="tblEmploye">
           <property name="styleSheet">
            <string notr="true">border-style:none;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>61</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>220</number>
                </property>
                <property name="bottomMargin">
                 <number>21</number>
                </property>
                <item>
                 <widget class="QPushButton" name="btnRefresh">
                  <property name="minimumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
border-radius:0px;
border-style: none;
background-color:#3737F9;
color:white;
}

QPushButton:hover {
background-color:#2323A4;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normalon>:/icons/res/icons/SVG/Refresh.svg</normalon>
                   </iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="shortcut">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnAdd">
                  <property name="minimumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
border-radius:0px;
border-style: none;
background-color:#09AF25;
color:white;
}

QPushButton:hover {
background-color:#0E8522;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normalon>:/icons/res/icons/SVG/Ajout.svg</normalon>
                   </iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="shortcut">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnDel">
                  <property name="minimumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
border-radius:0px;
border-style: none;
background-color:#FF0046;
color:white;
}

QPushButton:hover {
background-color:#CE0038;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normalon>:/icons/res/icons/SVG/Delete2.svg</normalon>
                   </iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="shortcut">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnMod">
                  <property name="minimumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>60</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
border-radius:0px;
border-style: none;
background-color:#F99237;
color:white;
}

QPushButton:hover {
background-color:#D27E35;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normalon>:/icons/res/icons/SVG/Modify.svg</normalon>
                   </iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="shortcut">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
                <property name="topMargin">
                 <number>100</number>
                </property>
                <property name="bottomMargin">
                 <number>24</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Nom du type d'appareil</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txfEmploye">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>480</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLineEdit{
	border: 2px solid rgb(112, 112, 112);
	border-radius: 15px;
	background-color: white;
	padding-left:20px;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>20</number>
              </property>
              <property name="bottomMargin">
               <number>100</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>10</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">padding-left:150px;
color:black;</string>
                </property>
                <property name="text">
                 <string>Lien image</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
                <property name="spacing">
                 <number>30</number>
                </property>
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="bottomMargin">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="maximumSize">
                   <size>
                    <width>250</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Image descriptive de l'appareil</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnQuitter_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Segoe UI</family>
                    <pointsize>16</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
border-radius:15px;
background-color: #1D43DB;
color:white;
}

QPushButton:hover{
background-color:#2323A4;
}
</string>
                  </property>
                  <property name="text">
                   <string>Rechercher...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>16</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius:22px;
background-color: rgb(209, 122, 90);
color:white;
}

QPushButton:hover{
background-color:rgb(176, 103, 76);
}
</string>
            </property>
            <property name="text">
             <string>Retour à l'interface de gestion</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
